qiita_ware/ebi.py

```diff
@@ -272,8 +271,8 @@ def generate_study_xml(self):
 
         Returns
         -------
-        str
-            string with study XML values
+        Element
+            Object with study XML values
```

Comment:
Actually, we commonly use `pd.DataFrame` instead of `pandas.DataFrame` or `np.DataFrame` instead of `numpy.DataFrame`.

So just `ET.Element` should be good enough.